apps/root/src/pages/earn/home/components/one-click-migration-modal/index.tsx

```diff
@@ -0,0 +1,67 @@
+import useAvailableDepositTokens, { FarmWithAvailableDepositTokens } from '@hooks/earn/useAvailableDepositTokens';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Modal } from 'ui-library';
+import OneClickMigrationOptionsContent from './options-content';
+import usePushToHistory from '@hooks/usePushToHistory';
+import useTrackEvent from '@hooks/useTrackEvent';
+
+interface OneClickMigrationModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const OneClickMigrationModal = ({ open, onClose }: OneClickMigrationModalProps) => {
+  const [step, setStep] = React.useState<'select-vaults' | 'confirm-migration'>('select-vaults');
```

Comment:
If you were me, you would ask for an enum here 😄

Reply:
Oooh good catch